Sources/Swift/Integrations/SessionReplay/SentryOnDemandReplay.swift

```diff
@@ -246,12 +246,58 @@ class SentryOnDemandReplay: NSObject, SentryReplayVideoMaker {
 
     private func createVideoSettings(width: CGFloat, height: CGFloat) -> [String: Any] {
         return [
```

Comment:
`m`: As these settings are crucial for SR, does it maybe make sense to add a test validating the dict this method returns, so we have to mess up the settings in two places to break SR?

Reply:
I was also considering this, but I would have to change the visibility of the function to make it testable. I guess it's worth it, so I'll change it to `internal` and write a test.